test/afterburner/afterburner.yaml

```diff
@@ -0,0 +1,21 @@
+Version: 1.8
+Logging:
+    default: INFO
+
+General:
+    Modus:         List
+    Time_Step_Mode: None
+    Delta_Time:    0.1
+    End_Time:      10000.0
+    Randomseed:    -1
+    Nevents:       100
+
+Output:
+    Particles:
+        Format:          ["Binary", "Oscar2013"]
```

Comment:
Why do we need also the Oscar2013 output?

Reply:
You are right, this is not necessary. I removed Oscar output.